### DISCUSSION TITLE
Load file from variable

### DISCUSSION THREAD

**Discussion by sweta1991** [google/model-viewer#2431] [Q&A]
I have used model-viewer in my angular 6 project. I am trying to load file with url stored in variable. I am getting file not found error. Even though the file is present. I can download the file but not preview using model-viewer.
Is there a way to render src using [src]="fileURL" like angular?

**Comment by shamsail** [comment]
You can get the model-viewer element and change it's 'src' in the javascript if you are not able to do so in the html directly as you were using in angular. Just get model-viewer element and change src where you are changing the fileURL variable value.

```
//get or change fileURL
let viewer = new ModelViewerElement();
viewer = document.getElementById("viewerID");
viewer.src = fileURL;
```
